src/PanoViewer/PanoViewer.js

```diff
@@ -486,13 +486,28 @@ export default class PanoViewer extends Component {
 		if (!this._isReady) {
 			return this;
 		}
-		this._width = (size && size.width) ||
-			parseInt(window.getComputedStyle(this._container).width, 10);
-		this._height = (size && size.height) ||
-									parseInt(window.getComputedStyle(this._container).height, 10);
-		this._aspectRatio = this._width / this._height;
-		this._photoSphereRenderer.updateViewportDimensions(this._width, this._height);
+
+		let containerSize;
+
+		if (!size !== undefined || size.width !== undefined || size.height !== undefined) {
```

Comment:
I think that  `updateViewportDimensions`  can make simple using destructuring assignment.
How about it?
```js
updateViewportDimensions(size={width:undefined, height:undefined}) {
	if (!this._isReady) {
		return this;
	}

	let containerSize;
	let [width, height] = size;

	if (width !== undefined || height !== undefined) {
		containerSize = window.getComputedStyle(this._container);
		width = parseInt(containerSize.width, 10);
		height = parseInt(containerSize.height, 10);
	}

	// Skip if viewport is not changed.
	if (width === this._width && height === this._height) {
		return this;
	}

	this._width = width;
	this._height = height;

	this._aspectRatio = width / height;
	this._photoSphereRenderer.updateViewportDimensions(width, height);
	this._yawPitchControl.option("aspectRatio", this._aspectRatio);
	this._yawPitchControl.updatePanScale({height});

	this.lookAt({}, 0);
	return this;
}
```

Reply:
@mixed Thank you for suggestion. That makes code more neat! 
I'll apply it except the some logical part. Because there's some difference that I've intended.